source/css/_objects.carousels.scss

```diff
@@ -158,3 +161,11 @@
     transform: translate(0, 0);
   }
 }
+.carousel-nav--4-3 {
```

Comment:
Correct. You can create another helper class that add padding for a 16:9 ratio. It'd look something like this:

```scss
@mixin carousel-nav--16-9() {
  .slick-dots {
    top: 0;
    bottom: auto;
    left: 0;
    padding: rem(10);
    padding-top: calc(56.25% - 28px); // 16:9 padding
    transform: translate(0, 0);
  }
}
```

See here for more on intrinsic ratios - http://alistapart.com/article/creating-intrinsic-ratios-for-video